src/components/common/TagForm/RegisteredTagList.tsx

```diff
@@ -11,32 +11,35 @@ function RegisteredTagList({
   onClick: (tag: TagType) => void;
 }) {
   return (
-    <section css={registeredTagsCss}>
-      <h2 css={userTagsTitleCss}>등록된 태그 목록</h2>
-      {Boolean(registeredTags.length) ? (
-        registeredTags.map(tag => (
-          <Tag
-            key={tag.id}
-            content={tag.content}
-            onClick={() => {
-              onClick(tag);
-            }}
-          />
-        ))
-      ) : (
-        <div css={notHaveUserTagsCss}>등록된 태그가 없습니다.</div>
-      )}
-    </section>
+    <>
+      <h2 css={userTagsTitleCss}>내 영감에 쓰인 태그</h2>
+      <section css={registeredTagsCss}>
+        {Boolean(registeredTags.length) ? (
+          registeredTags.map(tag => (
+            <Tag
+              key={tag.id}
+              content={tag.content}
+              onClick={() => {
+                onClick(tag);
+              }}
+            />
+          ))
+        ) : (
+          <div css={notHaveUserTagsCss}>등록된 태그가 없습니다.</div>
+        )}
```

Comment:
```suggestion
const isRegisteredTagExist = Boolean(registeredTags.length);

...
        { isRegisteredTagExist ? (
          registeredTags.map(tag => (
            <Tag
              key={tag.id}
              content={tag.content}
              onClick={() => {
                onClick(tag);
              }}
            />
          ))
        ) : (
          <div css={notHaveUserTagsCss}>등록된 태그가 없습니다.</div>
        )}
```

변수로 빼둬도 좋을 거 같아요 ! 물론 넘어가도 좋습니다 ~~ 👍 👍

Reply:
좋습니다!! [79b2102](https://github.com/depromeet/ygtang-client/pull/578/commits/79b2102fac230d3eb59354e3cff8bc52d2020dd0)